src/terraform_plan.sh

```diff
@@ -7,6 +7,9 @@ function terraformPlan {
   planExitCode=${?}
   planHasChanges=false
   planCommentStatus="Failed"
+  planOutputFile="${tfWorkingDir}/plan.txt"
```

Comment:
I think it would be better to either:

1. Write to a user-configurable path in the repo (and only write if explicitly requested)
2. Write to a temporary location outside the repo

For approach #2, 
```suggestion
  planOutputFile="$(mktemp)"
```

Reply:
Is there any harm in writing it in the checked out copy of the repo though?  The whole environment is destroyed after the job has finished so the file only exists for the duration of the run.  If you don't archive it, it's essentially a temp file.

Reply:
so I tried making this change but it looks like other steps in the workflow may not have access to the temp folder.  In this case, the upload artifact step errored:

```bash
Run actions/upload-artifact@v1
##[error]Path does not exist /tmp/tmp.lIogbf
```

Reply:
damn :'( there goes that idea